### PR TITLE
os: add pstore documentation

### PR DESCRIPTION
**Pull request by crawford**
Helps address https://github.com/coreos/bugs/issues/477.

### CODE REVIEW

File: os/collecting-crash-logs.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Panic#2 Part1
+
+$ head --lines=1 /sys/fs/pstore/dmesg-erst-6319986351055831044
+Panic#2 Part2
+
+$ head --lines=1 /sys/fs/pstore/dmesg-erst-6319986351055831045
+Panic#2 Part3
+```
+
+It is important to note that the pstore typically has very limited storage space (on the order of kilobytes) and will not overwrite entries when out of space. The files in `/sys/fs/pstore` must be removed to free up space. The typical approach is to move the files from the pstore to a more permanent storage location on boot, but CoreOS will not do this automatically for you.
```
Comment (robszumski) [inline os/collecting-crash-logs.md]: It would be great to give an example of how to do this, but not a blocker.

**Comment by robszumski** [comment]
LGTM

File: os/collecting-crash-logs.md (hunk 1 above)
Comment (cvanlabe) [inline os/collecting-crash-logs.md]: Like!

Comment (crawford) [inline os/collecting-crash-logs.md]: Darn. You should be able to capture the console output though, which will have panic traces.

**Pull request merged**